tap_hubspot/streams.py

```diff
@@ -93,3 +88,69 @@ def schema(self) -> dict:
         if self.cached_schema is None:
             self.cached_schema, self.properties = self.get_custom_schema()
         return self.cached_schema
+
+class PropertiesStream(HubspotStream):
+    """Define custom stream."""
+    schema_filepath = None
+    primary_keys = ["name"]
+    replication_key = "name"
+    schema = th.PropertiesList(
+                th.Property('name', th.StringType, required=True),
+                th.Property('updatedAt', th.DateTimeType, required=False),
+                th.Property('createdAt', th.StringType, required=False),
+                th.Property('label', th.StringType, required=False),
+                th.Property('type', th.StringType, required=False),
+                th.Property('fieldType', th.StringType, required=False),
+                th.Property('description', th.StringType, required=False),
+                th.Property('groupName', th.StringType, required=False),
+                th.Property('options',
+                    th.ArrayType(
+                        th.ObjectType(
+                            th.Property('label', th.StringType, required=False),
+                            th.Property('value', th.StringType, required=False),
+                            th.Property('displayOrder', th.IntegerType, required=False),
+                            th.Property('hidden', th.BooleanType, required=False),
+                        )
+                    ), required=False),
+                th.Property('displayOrder', th.IntegerType, required=False),
+                th.Property('calculated', th.BooleanType, required=False),
+                th.Property('externalOptions', th.BooleanType, required=False),
+                th.Property('hasUniqueValue', th.BooleanType, required=False),
+                th.Property('hasUniqueValue', th.BooleanType, required=False),
+                th.Property('hidden', th.BooleanType, required=False),
+                th.Property('hubspotDefined', th.BooleanType, required=False),
+                th.Property('modificationMetadata',
+                    th.ObjectType(
+                        th.Property('archivable', th.BooleanType, required=False),
+                        th.Property('readOnlyDefinition', th.BooleanType, required=False),
+                        th.Property('readOnlyOptions', th.BooleanType, required=False),
+                        th.Property('readOnlyValue', th.BooleanType, required=False),
+                    ), required=False),
+                th.Property('formField', th.BooleanType, required=False),
+            ).to_dict()
```

Comment:
I think I prefer having the schema in a separate file, that way it doesn't bloat this file as schemas can get pretty big, and the JSON format is very readable. What do you think?

Reply:
Fair. I was interested in trying it out, but it's definitely less clean